include/extractor/scripting_environment_lua.hpp

```diff
@@ -68,7 +68,14 @@ class LuaScriptingEnvironment final : public ScriptingEnvironment
                     tbb::concurrent_vector<boost::optional<InputRestrictionContainer>>
                         &resulting_restrictions) override;
 
+    void setExtractionContainers(ExtractionContainers &extractor) override;
+
+    // FIXME This is static because I don't know how to access to this in lua nodeRef class binding.
+    osmium::Location getLocation(osmium::NodeRef &nref);
```

Comment:
@daniel-j-h  I try this,
Now ExtractionContainers is non static and code compile.
But when i try to access to location in lua i have this error : 

> maxime@lemurien:~/DEV/osrm/osrm-backend$ build/osrm-extract ../pbf-data/andorra-latest.osm.pbf -t 1
> [info] Using script profile.lua
> [info] Input file: andorra-latest.osm.pbf
> [info] Profile: profile.lua
> [info] Threads: 1
> [STXXL-MSG] STXXL v1.4.1 (prerelease/Debug)
> [STXXL-ERRMSG] Warning: no config file found.
> [STXXL-ERRMSG] Using default disk configuration.
> [STXXL-MSG] Disk '/var/tmp/stxxl' is allocated, space: 1000 MiB, I/O implementation: syscall autogrow delete_on_exit queue=0 devid=0
> [info] Parsing in progress..
> [info] input file generated by Osmium (http://wiki.openstreetmap.org/wiki/Osmium)
> [info] timestamp: 2016-11-06T20:28:03Z
> [info] Found 3 turn restriction tags:
> [info]   motorcar
> [info]   motor_vehicle
> [info]   vehicle
> way id : 	5203906
>   node id  : 	3655224911
> terminate called after throwing an instance of 'osrm::util::exception'
>   what():  ERROR occurred in profile script:
> No matching overload found, candidates:
> Location location(custom [N4osrm9extractor23LuaScriptingEnvironmentE]&,NodeRef&)
> Abandon (core dumped)